tests/query/parser/test_expressions.py

```diff
@@ -9,6 +9,117 @@
 from snuba.query.parser.expressions import parse_aggregation
 
 test_data = [
+    (
+        ["f('2020/07/16')", None, None],
```

Comment:
your arithmetic expression parser does not deal with division anyway :D yet.
Please turn it into an actually possible arithmetic expression: `'2020-7-16'`

Reply:
I replaced "/" with the symbols currently supported in arithmetic: "+" and "*".